lx_administration/models/vault/manager_utils.py

```
@@ -34,14 +34,14 @@ def ensure_local_vault_key(path: Union[Path, str], logger=None):
         generate_ansible_key(local_vault_key)
 
 
-def generate_ansible_key(key_path: Path, mode="password"):
+def generate_ansible_key(key_path: Path, mode: Literal["password", "passphrase"] = "password"):
```

Ruff (0.12.2)

Line too long (95 > 88)

(E501)